src/screens/CreatePost/elements/UserProfile.js

```diff
@@ -55,7 +56,8 @@ const UserProfile = ({
   photo,
   onPress,
   isAnonymous = true,
-  anonUserInfo = null
+  anonUserInfo = null,
+  isToggleDisabled = false
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new prop `isToggleDisabled` has been added to the component. Ensure that this prop is being passed correctly from the parent component and that it's being used as intended.
```diff
-  anonUserInfo = null
+  anonUserInfo = null,
+  isToggleDisabled = false
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->